From c69d1b2eb1cf2fe324fef0121b583803435d469c Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 26 Jan 2018 22:23:03 +0100
Subject: [PATCH 19/19] FingerprintEnrollFindSensor: Don't overlay front for
 side sensors

* Both the base image and the base image + overlay are equally nonsense
  for devices with a side fingerprint scanner, but for those that overlay
  a custom fingerprint enrollment graphic, this image overlay wreaks havoc
* Only set this overlay to be visible if the sensor is on the front, to
  allow devices with side sensors to only overlay the base image

Change-Id: I7cbcea4830d4526cf9f7d139e7f84f21117fa9c4
---
 .../settings/fingerprint/FingerprintEnrollFindSensor.java       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/fingerprint/FingerprintEnrollFindSensor.java b/src/com/android/settings/fingerprint/FingerprintEnrollFindSensor.java
index c392783122..25ee1095fa 100644
--- a/src/com/android/settings/fingerprint/FingerprintEnrollFindSensor.java
+++ b/src/com/android/settings/fingerprint/FingerprintEnrollFindSensor.java
@@ -85,7 +85,7 @@ public class FingerprintEnrollFindSensor extends FingerprintEnrollBase {
         message.setText(getString(
                 R.string.security_settings_fingerprint_enroll_find_sensor_message_cm,
                 location));
-        if (sensorLocation != SENSOR_LOCATION_BACK) {
+        if (sensorLocation == SENSOR_LOCATION_FRONT) {
             findViewById(R.id.fingerprint_sensor_location_front_overlay)
                     .setVisibility(View.VISIBLE);
         }
-- 
2.17.1

